packages/bigtable/test/index.js

```diff
@@ -106,12 +106,27 @@ describe('Bigtable', function() {
       fakeUtil.normalizeArguments = normalizeArguments;
     });
 
+    it('should throw if a cluster is not provided', function() {
+      assert.throws(function() {
+        new Bigtable();
```

Comment:
Not sure why, but coveralls is claiming this is untested. Could we try changing this to:

```js
new Bigtable({
  zone: ZONE
});
```